lnbits/wallets/boltz.py

```diff
@@ -95,7 +95,8 @@ async def create_invoice(
             wallet_id=self.wallet_id,
             accept_zero_conf=True,
             external_pay=True,
-            description=memo,
+            # boltz rejects nbsp char (produced by JS Intl.NumberFormat api)
+            description=memo.replace(" ", " "),  # noqa: RUF001
```

Comment:
it looks like space char is replaced by space char?
am I not seeing something right?

Reply:
Browser, renders it the same way, but its a different char under the hood.
For example, pycharm:
![image](https://github.com/user-attachments/assets/5cffb4c0-6438-4825-88c9-5d07c419e2cd)